background/background.js

```diff
@@ -25,6 +29,7 @@ const mainAdditionalSearchEngineMenuItemId = 'mainAdditionalSearchEngine';
 const yahooMenuItemId = 'yahoo';
 const yahooJapanMenuItemId = 'yahooJapan';
 const hostPermissions = { origins: ['*://*/*'] };
+const backNavigatableUrls = { urls: ['*://*/*'], types: ['main_frame'] };
```

Comment:
Add 'tabId: -1' to the filter. Without this filter option, the event listener is called when a tab loads a page.

Reply:
6566d78